src/main/java/com/atlassian/db/replica/internal/SQLOpeartionType.java

```diff
@@ -0,0 +1,50 @@
+package com.atlassian.db.replica.internal;
+
+public class SQLOpeartionType {
```

Comment:
```suggestion
public class SQLOperationType {
```
There are more "opeartion " typos in the PR